src/index.ts

```diff
@@ -29,14 +29,16 @@ async function getPixels(data: Uint8Array, mimeType: string): Promise<NdArray<Ui
  * the necessary support in Canvas 2D.
  *
  * @param pixels ndarray of shape W x H x 4.
- * @param mimeType `image/jpeg`, `image/png`, etc.
+ * @param mimeType `image/jpeg`, `image/png`, `image/webp` etc.
+ * @param quality quality as a number from 0 to 1, inclusive
  * @returns
  */
 async function savePixels(
 	pixels: NdArray<Uint8Array | Uint8ClampedArray>,
-	mimeType: string
+	mimeType: string,
+	quality?: number
```

Comment:
I think I'd slightly prefer that the third argument be an options object, with quality as a property, for the public API. No preference on the internal API!

Reply:
So the reason I didn't do it that way is because I thought it was a confusing halfway between the several DOM methods:

- [`canvas.toDataURL([ type [, quality ] ])`](https://html.spec.whatwg.org/multipage/canvas.html#dom-canvas-todataurl)
- [`canvas.toBlob(callback [, type [, quality ] ])`](https://html.spec.whatwg.org/multipage/canvas.html#dom-canvas-toblob)
- [`offscreenCanvas.convertToBlob(optional ImageEncodeOptions options = {})`](https://html.spec.whatwg.org/multipage/canvas.html#dom-offscreencanvas-converttoblob)

And it felt awkward to wrap essentially one argument in an object. But I think it makes sense to evolve the API by allowing the second argument to be a `string | {type?:string, quality?:number}` so I'll change it to that.

Reply:
> But I think it makes sense to evolve the API by allowing the second argument to be a string | {type?:string, quality?:number} so I'll change it to that.

That works for me! A future major release could switch to the offscreen-canvas-like API.